huaweicloud/services/dcs/resource_huaweicloud_dcs_instance.go

```diff
@@ -586,6 +586,14 @@ func resourceDcsInstancesCreate(ctx context.Context, d *schema.ResourceData, met
 			return fmtp.DiagErrorf("[DEBUG] Error the order is not completed while "+
 				"creating DCS instance. %s : %#v", d.Id(), err)
 		}
+		err = common.WaitOrderComplete(ctx, bssClient, r.OrderId, d.Timeout(schema.TimeoutCreate))
+		if err != nil {
+			return diag.Errorf("the order is not completed while creating CBR vault: %v", err)
```

Comment:
the error message is  not correct

Reply:
all incorrect message have been modified.